WORKSPACE

```diff
@@ -0,0 +1,162 @@
+workspace(name = "pytorch")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("//tools/rules:workspace.bzl", "new_patched_local_repository")
+
+http_archive(
+    name = "bazel_skylib",
+    urls = [
+        "https://github.com/bazelbuild/bazel-skylib/releases/download/1.0.2/bazel-skylib-1.0.2.tar.gz",
+    ],
+)
+
+http_archive(
+    name = "com_google_googletest",
+    strip_prefix = "googletest-cd6b9ae3243985d4dc725abd513a874ab4161f3e",
+    urls = [
+        "https://github.com/google/googletest/archive/cd6b9ae3243985d4dc725abd513a874ab4161f3e.tar.gz",
+    ],
+)
+
+new_local_repository(
+    name = "pybind11",
+    build_file = "//third_party:pybind11.BUILD",
+    path = "third_party/pybind11",
+)
+
+http_archive(
+    name = "python3",
+    build_file = "//third_party:python3.BUILD",
+    urls = ["https://artifacts.int.uberatc.com/artifactory/auto-bazel/38dd863d274a6f6e37264ffce5e0da76e70affa06451245167b1fcd3b93cbec3/python3.tar.gz"],
```

Comment:
These look like uber internal resources, is this supposed to be accessible from open source?

Reply:
Too right. I'll get this removed.